app/participants/admitting.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+from app.participants.knights import Knights
+
+
+def creating_knights(knight_config: dict) -> Knights:

```

Comment:
Do not use gerund or single-standing nouns as your method name here and in other methods, use verbs instead.